docs/CHANGELOG.asciidoc

```diff
@@ -50,6 +50,33 @@ To review the breaking changes in previous versions, refer to the following:
 <<breaking-changes-8.0.0-rc1,8.0.0-rc1>> | <<breaking-changes-8.0.0-beta1,8.0.0-beta1>> | <<breaking-changes-8.0.0-alpha2,8.0.0-alpha2>> | 
 <<breaking-changes-8.0.0-alpha1,8.0.0-alpha1>>
  
+[float]
+[[known-issue-8.0.0-rc2]]
+=== Known issue
+
+[discrete]
+[[known-issue-123550]]
+.Importing or copying saved objects in 8.0.0 can cause "weak links" to break
+[%collapsible]
+====
+*Details* +
+Sometimes users add "weak links" in saved objects -- for example, a dashboard may include a Markdown panel that contains a relative URL to
+another dashboard. This is considered a weak link because it is not defined by the saved object's relationships, rather it is defined by a
+user in free text. Such weak links can break if **both** of the following conditions are true:
+
+1. You are importing the saved object(s) into multiple spaces, _OR_ you are copying the saved object(s) into another space
+2. The saved object(s) did not already exist in the destination(s) prior to upgrading to the 8.0.0 release
+
+This problem occurs because <<saved-object-ids-impact-when-using-import-and-copy,saved object IDs can change during import/copy starting in
+8.0>>. We are planning to address this issue in a bugfix for the 8.0.1 and 8.1 releases. For more information, refer to
+{kibana-pull}123550[#123550]. If you are impacted, you will be able to re-import (or re-copy) your saved objects after the bugfix is
+implemented to preserve these weak links.
+
+*Impact* +
+Saved objects that existed in 7.x and are migrated upon upgrade will **not** be impacted.
+Only _new_ saved objects that are imported/copied _multiple times_ will be impacted.
```

Comment:
What if I take an export from a 7.x cluster, and then import into a fresh 8.0 cluster? I was under the impression that this would also break weak links -- is that accurate? If so, then the `multiple times` clause here isn't entirely accurate

Reply:
> What if I take an export from a 7.x cluster, and then import into a fresh 8.0 cluster? I was under the impression that this would also break weak links -- is that accurate?

Yes! Only if you import them more than once, though.

The first time you import objects into a fresh cluster, the IDs won't change. The _second_ time you import them (into a different space), the IDs will change.

Sounds like this sentence isn't clear enough. How about I change it to this?

"Only _new_ saved objects that are imported or copied _multiple times_ (causing object IDs to change) are be impacted."

Reply:
Either version is fine, I just misunderstood the impact. Thanks for clarifying!